cordova-lib/src/cordova/plugin.js

```diff
@@ -305,6 +289,50 @@ module.exports = function plugin(command, targets, opts) {
     });
 };
 
+function determinePluginTarget(projectRoot, cfg, target, fetchOptions) {
+    var parts = target.split('@');
+    var id = parts[0];
+    var version = parts[1];
+
+    // If no version is specified, retrieve the version (or source) from config.xml
+    if (!version && !cordova_util.isUrl(id) && !cordova_util.isDirectory(id)) {
```

Comment:
I'd rather reverse `if` condition here to return `target` immediately if there is no need for any processing. This is just a matter of taste of course but IMO it would help to improve code readability a bit. The same for L302 and L304